js/public/bindings.js

```diff
@@ -69,60 +73,100 @@ const addBinding = values => {
     }
     // Basically a form but hush
     const binding = document.createElement('div')
-    binding.className = "binding row"
+    binding.className = 'binding row'
     // Select for input type
-    const bindingType = labelInput({ label: "Type", name: 'inputType', elem: 'select', onChange: setBinding })
+    const bindingType = labelInput({
```

Comment:
Expanding those object notations makes the file much longer which counters the added legibility.
Only case this is done is for the "bindingInvert" as it's a different type of input (checkbox) with its own "onChange" definition (see below)